corehq/apps/users/models.py

```diff
@@ -382,6 +382,7 @@ class DomainMembership(Membership):
     override_global_tz = BooleanProperty(default=False)
     role_id = StringProperty()
     location_id = StringProperty()
+    assigned_location_ids = StringListProperty()
```

Comment:
Maybe good to confirm, but it's probably best to do it now, while web users' locations aren't used for much.